ml-agents/mlagents/trainers/settings.py

```diff
@@ -119,6 +121,7 @@ def _reward_signal_steps_per_update_default(self):
         return self.steps_per_update
 
 
+# INTRINSIC REWARD SIGNALS #############################################################
```

Comment:
This was an attempt at organizing the sections of this file better. I can remove if it does not deliver.